config/config.go

```diff
@@ -0,0 +1,63 @@
+package config
+
+import (
+	"fmt"
+
+	"github.com/cozy/swift"
+	"github.com/spf13/viper"
+)
+
+var config *Config
+
+type Config struct {
+	SwiftConnection *swift.Connection
+}
+
+func NewConfig() (*Config, error) {
```

Comment:
[nit] The convention for go packages is to use `New` and not `NewFoo` when the package is named `foo` and has a single exported `Foo` struct.